src/components/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -123,22 +123,22 @@ describe('<ThemeProvider />', () => {
 
   it('sets color system properties when global theming is enabled', () => {
     const themeProvider = mountWithAppProvider(
-      <ThemeProvider theme={{}}>
+      <ThemeProvider theme={{UNSTABLE_colors: {surface: '#ffffff'}}}>
```

Comment:
I'm a little concerned about why I had to pass a theme to make this test pass. I might have screwed up how defaults are applied.